assets/css/_sass/website/_mixins.scss

```diff
@@ -11,6 +11,7 @@
 @mixin animation($animate) {
   -webkit-animation: $animate;
      -moz-animation: $animate;
+       -o-animation: $animate;
```

Comment:
vendor prefix is unnecessary. Opera hasn't used the prefix since version 15 (current version is 52)